cfgov/search/elasticsearch_helpers.py

```diff
@@ -49,3 +51,10 @@ def get_synonyms():
         synonynm_filter,
         'lowercase'
     ])
+
+
+def environment_specific_index(base_name):
+    if settings.DEPLOY_ENVIRONMENT == 'production' or settings.DEPLOY_ENVIRONMENT == 'local':  # noqa: E501
+        return base_name
+    else:
+        return f'{settings.DEPLOY_ENVIRONMENT}-{base_name}'
```

Comment:
If `settings.DEPLOY_ENVIRONMENT` is unset, should this return the base name without the leading dash?

Reply:
I would think so